SingularityService/src/test/java/com/hubspot/singularity/scheduler/SingularitySchedulerTest.java

```diff
@@ -3424,6 +3424,61 @@ public void testRequestedPorts() {
     Assertions.assertEquals(1, taskManager.getActiveTaskIds().size());
   }
 
+  @Test
+  public void testPortUsage() {
```

Comment:
this test doesnt seem related to the PR -- can we move it to a separate one if it's still necessary?